src/deps_compat.jl

```diff
@@ -9,12 +9,43 @@ each package listed under `deps`.
   them.
 
 # Keyword Arguments
+## Test choosers
+- `check_extras = false`: If true, additionally check "extras". A NamedTuple
+  can be used to pass keyword arguments with test options (see below).
+- `check_weakdeps = false`: If true, additionally check "weakdeps". A NamedTuple
+  can be used to pass keyword arguments with test options (see below).
+
+## Test options
+If these keyword arguments are set directly, they only apply to the standard test
```

Comment:
Can you think of a reason why you wouldn't want the same set of kwargs for all your deps checks?